README.md

```diff
@@ -1,40 +1,249 @@
 
-Portfolio
+Modify this file to match your project and remove sections that don't apply.
 
-## Link Portfolio
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-- Link: https://jlberbesi.github.io/Portfolio/
+OPTIONAL SECTIONS:
+- FAQ
 
-## Built With
+After you're finished please remove all the comments and instructions!
+-->
 
-- HTML
-- CSS
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-## Getting Started
+  <h3><b>Microverse README Template</b></h3>
 
-Project only contains a HTML  and  css files. No prerequisites are required nor any other special steps to install it. 
+</div>
 
-## Authors
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Jose Berbesí Portafolio] <a name="about-project"></a>
+
+Welcome to my software developer portfolio! Here, you'll find a collection of my best web application projects. My portfolio showcases my skills in various programming languages and frameworks, such as Python, React, and Angular. Feel free to take a look at my projects and explore my experience in web application development.
+**[Jose Berbesí Portafolio]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+For this project, I haven't used complex technologies. I mainly used HTML and CSS for layout design.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Html</a></li>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+There are no features currently built as this is currently just an informational site.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+NA
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```

Comment:
- [OPTIONAL] Please consider removing all the comments that starts with ">" from your README file since they are just a guide 😄